### PR TITLE
draft a blob post

### PR DESCRIPTION
**Pull request by r10s**
~~todo: add a screenshot for disappearing messages.~~

~~todo: what's missed?~~

### CODE REVIEW

**Comment by Jikstra** [comment]
Screenshot_20200731_221448


Maybe this helps?

File: _posts/2020-07-30-summer-update.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: Disappearing messages and Video chats
+author: bjoern
+image: ../assets/blog/2020-07-scan-videochat-provider.jpg
+---
+
+We're happy that we can introduce **two more major, frequently requested features** in Delta Chat:
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `We're happy to introduce ...`

File: _posts/2020-07-30-summer-update.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+_Disappearing messages_ and _Video chats_.
+
+
+### Video chats
+
+_"How can e-mail and video chats go together?"_
+
+_"And what if the called person does not even use Delta Chat?"_
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `And what happens if the called person doesn't even use Delta Chat?`

File: _posts/2020-07-30-summer-update.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+_"How can e-mail and video chats go together?"_
+
+_"And what if the called person does not even use Delta Chat?"_  
+(remember, we do not force people to use our app)
+
+_"Also, how this can fit with our decentralized approach?"_
+
+We were thinking a lot about these questions in the past,
+together with different user groups and tester,
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `testers`

File: _posts/2020-07-30-summer-update.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+and driven by various ux-researches.
+
+
+<img src="../assets/blog/2020-07-scan-videochat-provider.jpg" width="320" style="float:right; clear:both; margin-left:1em; margin-bottom:.2em;" alt="" />
+So - it is **up to the user to decide which video chat provider to use.**
+
+- They can enter it in the **settings**
+- The video provider may come from the **provider-info** with the other server-data
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `with other server-data`

File: _posts/2020-07-30-summer-update.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+together with different user groups and tester,
+and driven by various ux-researches.
+
+
+<img src="../assets/blog/2020-07-scan-videochat-provider.jpg" width="320" style="float:right; clear:both; margin-left:1em; margin-bottom:.2em;" alt="" />
+So - it is **up to the user to decide which video chat provider to use.**
+
+- They can enter it in the **settings**
+- The video provider may come from the **provider-info** with the other server-data 
+(which makes a lot of sense, however, currently this is not supported by many email-provider).
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `which makes a lot of sense, however this is currently not supported by many email-providers`

File: _posts/2020-07-30-summer-update.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+- Finally, the user can also scan a video chat provider **from a qr-code** - 
+this way, video chat providers can be shared easily and quickly.
+
+Delta Chat is **not very picky** about the provider choosen -
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `Delta Chat is  not very picky about the chosen provider`

File: _posts/2020-07-30-summer-update.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+the user can use every instance that reads the roomname from the url, 
+which is basically any open service, jitsy, talky, appear.in and many more,
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `The user can use every instance that contains the roomname in the url, which includes famous services as jitsi, talky.io, appear.in and many more.`

Would skip the `may they are open or not` part.

File: _posts/2020-07-30-summer-update.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+may they are open or not.
+
+<img src="../assets/blog/2020-07-videochat-invite2.png" width="220" style="float:right; clear:both; margin-left:1em; margin-bottom:.2em;" alt="" />
+We believe in widely decentralizing the video chat services
+is a good idea for overall privacy.
+Giving the user the option to just use existing instances helps a lot on this approach.
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `Enabling the user to use any existing instance or hosting their own helps a lot for this approach.`

File: _posts/2020-07-30-summer-update.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+However, once set, **starting a video chat is simple** -
+just tap "Video chat" in any one-to-one chat and wait for the recipient to join.
+(there is no need for the recipient to set a video chat provider on its own -
+and also, the recipeint is still free to join the chat from a
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `recipient`

File: _posts/2020-07-30-summer-update.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+and also, the recipeint is still free to join the chat from a 
+non-delta-client, if they really want to :)
+
+_Video chats can be started from Android 1.12 and Desktop 1.10 and
+can be joined from all existing clients_
+
+## Disappearing messages
+
+
+Another major feature are of all current Delta Chat versions are the so called
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `Another major feature in all recent Delta Chat versions on all platforms are the so called`

File: _posts/2020-07-30-summer-update.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+## Disappearing messages
+
+
+Another major feature are of all current Delta Chat versions are the so called 
+_Disappearing messages_.
+
+Once enabled in the settings,
+with that, one user of a chat can decide that he likes to see all subsequent
+messages disappearing.
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: Once the feature is enabled in the settings, any user can decide to see all subsequent messages disappearing.

File: _posts/2020-07-30-summer-update.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+After a chosen timeout,
+the messages will disappear on all the users devices, 
+on the users server -
+and moreover, also on devices and servers of 
+the other memvers if they are using Delta Chat and did not make a copy somehow.
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: `members` `copy/screenshot`

File: _posts/2020-07-30-summer-update.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+
+It is all about trust -
+you cannot guarantee deletion in any way, 
+not for Delta Chat nor for any other messengers -
+eg. users may use use a camera ...
```
Comment (Jikstra) [inline _posts/2020-07-30-summer-update.md]: users may take a screenshot of the message or just write it down on piece of paper using this retro technology called pen.

**Review by Jikstra — COMMENTED** [review]
Some ideas

File: _posts/2020-07-30-summer-update.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+Summer is there - and also a **new round of Delta Chat releases**.
+
+We're happy that we could fix tons of bugs the last weeks,
+so that the overall experience of Delta Chat will be a much more stable one.
+Also we added lots of smaller, but nevertheless important things.
+
+- Sending an receiving messages is more reliable
+- Desktop's message got several long awaited fixes
+- Windows and Linux now support more emoji 🥳
+- camera handling updated on iOS
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
- Camera handling updated on iOS
```

File: _posts/2020-07-30-summer-update.md [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+- enlarge profile image on iOS ↕️
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
- Enlarge profile image on iOS ↕️
```
Embiggen

File: _posts/2020-07-30-summer-update.md [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+- iOS profile view got an update
+- show a warning when the password was changed on the server
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
- Show warning when password changed on the server
```

File: _posts/2020-07-30-summer-update.md [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+- improve error handling when there is no network
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
- Improved offline error handling
```

File: _posts/2020-07-30-summer-update.md [hunk 18]
```diff
@@ -0,0 +1,113 @@
...
+introduce two more major, frequently requested features in Delta Chat:
+_Disappearing messages_ and _Video chats_.
+
+
+### Video chats
+
+_"How can e-mail and video chats go together?"_
+
+_"And what happens if the called person doesn't even use Delta Chat?"_  
+(remember, we do not force people to use our app)
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
(Remember, we do not force use of our app on anyone.)
```

File: _posts/2020-07-30-summer-update.md [hunk 19]
```diff
@@ -0,0 +1,113 @@
...
+
+_"Also, how this can fit with our decentralized approach?"_
+
+We were thinking a lot about these questions in the past,
+together with different user groups and testers,
+and driven by various ux-researches.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
and driven by various UX-research.
```

File: _posts/2020-07-30-summer-update.md [hunk 20]
```diff
@@ -0,0 +1,113 @@
...
+
+
+<img src="../assets/blog/2020-07-scan-videochat-provider.jpg" width="320" style="float:right; clear:both; margin-left:1em; margin-bottom:.2em;" alt="" />
+So - it is **up to the user to decide which video chat provider to use.**
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:42]:
```suggestion
So—it is **up to the user to decide which video-chat provider to use.**
```

File: _posts/2020-07-30-summer-update.md [hunk 21]
```diff
@@ -0,0 +1,113 @@
...
+
+- They can enter it in the **settings**
+- The video provider may come from the **provider-info** with other server-data 
+(which makes a lot of sense, however this is currently not supported by many email-providers).
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
(which makes a lot of sense, however this is currently not supported by many e-mail providers).
```

File: _posts/2020-07-30-summer-update.md [hunk 22]
```diff
@@ -0,0 +1,113 @@
...
+- Finally, the user can also scan a video chat provider **from a qr-code** -
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
- Finally, the user can also scan a video-chat provider **from a QR-code** - 
```

File: _posts/2020-07-30-summer-update.md [hunk 23]
```diff
@@ -0,0 +1,113 @@
...
+So - it is **up to the user to decide which video chat provider to use.**
+
+- They can enter it in the **settings**
+- The video provider may come from the **provider-info** with other server-data 
+(which makes a lot of sense, however this is currently not supported by many email-providers).
+- Finally, the user can also scan a video chat provider **from a qr-code** - 
+this way, video chat providers can be shared easily and quickly.
+
+Delta Chat is not very picky about the chosen provider:
+The user can use every instance that contains the roomname in the url, which includes famous
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
The user can use every instance that contains the room-name in the URL, which includes famous
```

File: _posts/2020-07-30-summer-update.md [hunk 24]
```diff
@@ -0,0 +1,113 @@
...
+services as jitsi, talky.io, appear.in and many more.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:52]:
```suggestion
services as Jitsi, talky.io, appear.in and many more.
```

File: _posts/2020-07-30-summer-update.md [hunk 25]
```diff
@@ -0,0 +1,113 @@
...
+
+<img src="../assets/blog/2020-07-videochat-invite2.png" width="220" style="float:right; clear:both; margin-left:1em; margin-bottom:.2em;" alt="" />
+We believe in widely decentralizing the video chat services
+is a good idea for overall privacy.
+Enabling the user to use any existing instance or hosting their own helps a lot for this approach.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:57]:
```suggestion
Letting the user pick any existing instance or hosting their own helps a lot with this approach.
```

File: _posts/2020-07-30-summer-update.md [hunk 26]
```diff
@@ -0,0 +1,113 @@
...
+
+However, once set, **starting a video chat is simple** -
+just tap "Video chat" in any one-to-one chat and wait for the recipient to join.
+(there is no need for the recipient to set a video chat provider on its own -
+and also, the recipient is still free to join the chat from a
+non-delta-client, if they really want to :)
+
+Finally, there is also the project [basicwebrtc](https://github.com/cracker0dks/basicwebrtc)
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
Finally, there is also the project [basicWebRTC](https://github.com/cracker0dks/basicwebrtc)
```

File: _posts/2020-07-30-summer-update.md [hunk 27]
```diff
@@ -0,0 +1,113 @@
...
+that introduces a slim, fast webrtc-experience.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
that introduces a slim, fast WebRTC-experience.
```

File: _posts/2020-07-30-summer-update.md [hunk 28]
```diff
@@ -0,0 +1,113 @@
...
+If a _basicwebrtc_ is detected as a videochat instance,
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
If _basicWebRTC_ is detected as a video-chat instance,
```

File: _posts/2020-07-30-summer-update.md [hunk 29]
```diff
@@ -0,0 +1,113 @@
...
+on desktop, the call is handled directly in-app.
+
+All in all, video chats are already pretty usable,
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:70]:
```suggestion
All in all, video-chats are already pretty usable,
```

File: _posts/2020-07-30-summer-update.md [hunk 30]
```diff
@@ -0,0 +1,113 @@
...
+however, we expect things being fine-tuned in the next releases.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:71]:
```suggestion
however, expect fine-tuning in the next few releases.
```

File: _posts/2020-07-30-summer-update.md [hunk 31]
```diff
@@ -0,0 +1,113 @@
...
+
+_Video chats can be started from Android 1.12 and Desktop 1.10.4 and
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:73]:
```suggestion
_Video-chats can be started from Android 1.12 and Desktop 1.10.4 and
```

File: _posts/2020-07-30-summer-update.md [hunk 32]
```diff
@@ -0,0 +1,113 @@
...
+can be joined from all existing clients_
+
+
+## Disappearing messages
+
+Another new feature in all recent Delta Chat versions on all platforms are the so called
+_Disappearing messages_.
+
+<img src="../assets/blog/2020-07-disappearing-options.png" width="250" style="float:left; clear:both; margin-right:1em; margin-bottom:.2em;" alt="" />
+Once the feature is enabled in the settings,
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:83]:
```suggestion
Once the feature is turned on in the settings,
```

File: _posts/2020-07-30-summer-update.md [hunk 33]
```diff
@@ -0,0 +1,113 @@
...
+After a chosen timeout,
+the **messages will disappear** on all the users devices,
+on the users server -
+and moreover, also on devices and servers of 
+the other members if they are using Delta Chat and did not make a copy or screenshot.
+
+**It is all about trust** -
+you cannot guarantee deletion in any way, 
+not for Delta Chat nor for any other messengers -
+eg. users may use a camera
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
e.g. users may use a camera
```

File: _posts/2020-07-30-summer-update.md [hunk 34]
```diff
@@ -0,0 +1,113 @@
...
+or just write it down on piece of paper using this retro technology called pen.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
or just write it down on piece of paper using pen technology.
```

File: _posts/2020-07-30-summer-update.md [hunk 35]
```diff
@@ -0,0 +1,113 @@
...
+and will offer the disappearing messages options by default then.
+
+
+## Try out the new releases!
+
+If you have not got the newest Delta Chat yet,
+see [get.delta.chat](https://get.delta.chat) for an overview.
+You'll also find **detailed changelogs** there.
+
+As usual, the different stores take different lengths of times for updates — thanks for your patience.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:111]:
```suggestion
As usual, the different stores will see the new version at different times—thanks for your patience.
```

File: _posts/2020-07-30-summer-update.md [hunk 36]
```diff
@@ -0,0 +1,113 @@
...
+
+
+## Try out the new releases!
+
+If you have not got the newest Delta Chat yet,
+see [get.delta.chat](https://get.delta.chat) for an overview.
+You'll also find **detailed changelogs** there.
+
+As usual, the different stores take different lengths of times for updates — thanks for your patience. 
+Also, thanks to all the ux-research attendees, testers, translators and developers to make this release happen :)
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
Also, thanks to all the UX-research attendees, testers, translators and developers to make this release happen :)
```

File: _posts/2020-07-30-summer-update.md [hunk 37]
```diff
@@ -0,0 +1,113 @@
...
+
+
+## Disappearing messages
+
+Another new feature in all recent Delta Chat versions on all platforms are the so called
+_Disappearing messages_.
+
+<img src="../assets/blog/2020-07-disappearing-options.png" width="250" style="float:left; clear:both; margin-right:1em; margin-bottom:.2em;" alt="" />
+Once the feature is enabled in the settings,
+any user can decide to see all subsequent messages disappearing.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:84]:
```suggestion
any user can decide to see all messages subsequently disappearing.
```
?

File: _posts/2020-07-30-summer-update.md [hunk 38]
```diff
@@ -0,0 +1,113 @@
+---
+title: Disappearing messages and Video chats
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
title: Disappearing Messages and Video-chats
```

File: _posts/2020-07-30-summer-update.md [hunk 39]
```diff
@@ -0,0 +1,113 @@
...
+---
+
+Summer is there - and also a **new round of Delta Chat releases**.
+
+We're happy that we could fix tons of bugs the last weeks,
+so that the overall experience of Delta Chat will be a much more stable one.
+Also we added lots of smaller, but nevertheless important things.
+
+- Sending an receiving messages is more reliable
+- Desktop's message got several long awaited fixes
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]: Desktop had a message?

File: _posts/2020-07-30-summer-update.md [hunk 40]
```diff
@@ -0,0 +1,113 @@
...
+- Windows and Linux now support more emoji 🥳
+- camera handling updated on iOS
+- enlarge profile image on iOS ↕️
+- iOS profile view got an update
+- show a warning when the password was changed on the server
+- improve error handling when there is no network
+
+Moreover, we're happy to
+introduce two more major, frequently requested features in Delta Chat:
+_Disappearing messages_ and _Video chats_.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md]:
```suggestion
_Disappearing messages_ and _Video-chats_.
```

File: _posts/2020-07-30-summer-update.md [hunk 41]
```diff
@@ -0,0 +1,113 @@
...
+
+
+### Video chats
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:28]:
```suggestion
### Video-chats
```

File: _posts/2020-07-30-summer-update.md [hunk 42]
```diff
@@ -0,0 +1,113 @@
...
+
+_"How can e-mail and video chats go together?"_
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:30]:
```suggestion
_"How can e-mail and video-chat go together?"_
```

File: _posts/2020-07-30-summer-update.md [hunk 43]
```diff
@@ -0,0 +1,113 @@
...
+
+
+<img src="../assets/blog/2020-07-scan-videochat-provider.jpg" width="320" style="float:right; clear:both; margin-left:1em; margin-bottom:.2em;" alt="" />
+So - it is **up to the user to decide which video chat provider to use.**
+
+- They can enter it in the **settings**
+- The video provider may come from the **provider-info** with other server-data 
+(which makes a lot of sense, however this is currently not supported by many email-providers).
+- Finally, the user can also scan a video chat provider **from a qr-code** - 
+this way, video chat providers can be shared easily and quickly.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:48]:
```suggestion
this way, video-chat providers can be shared easily and quickly.
```

File: _posts/2020-07-30-summer-update.md [hunk 44]
```diff
@@ -0,0 +1,113 @@
...
+
+Delta Chat is not very picky about the chosen provider:
+The user can use every instance that contains the roomname in the url, which includes famous
+services as jitsi, talky.io, appear.in and many more.
+
+<img src="../assets/blog/2020-07-videochat-invite2.png" width="220" style="float:right; clear:both; margin-left:1em; margin-bottom:.2em;" alt="" />
+We believe in widely decentralizing the video chat services
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:55]:
```suggestion
We believe in widely decentralizing the video-chat services
```

File: _posts/2020-07-30-summer-update.md [hunk 45]
```diff
@@ -0,0 +1,113 @@
...
+is a good idea for overall privacy.
+Enabling the user to use any existing instance or hosting their own helps a lot for this approach.
+
+However, once set, **starting a video chat is simple** -
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:59]:
```suggestion
However, once set, **starting a video-chat is simple** -
```

File: _posts/2020-07-30-summer-update.md [hunk 46]
```diff
@@ -0,0 +1,113 @@
...
+just tap "Video chat" in any one-to-one chat and wait for the recipient to join.
```
Comment (comradekingu) [inline _posts/2020-07-30-summer-update.md:60]:
```suggestion
just tap "Video-chat" in any one-to-one chat and wait for the recipient to join.
```

File: _posts/2020-07-30-summer-update.md (hunk 39 above)
Comment (r10s) [reply _posts/2020-07-30-summer-update.md]: there is a word missing :) it's desktop's message list

Comment (r10s) [reply _posts/2020-07-30-summer-update.md]: i will fix that.

File: _posts/2020-07-30-summer-update.md (hunk 37 above)
Comment (r10s) [reply _posts/2020-07-30-summer-update.md:84]: _"all subsequent messages" will disappear_ when the function is enabled.

however, at least is would read the suggestion as: _"all messages" disappear subsequently._  - which is not the same.

**Comment by r10s** [comment]
thanks a lot for all the corrections and suggestions, i incorporated most of them, i think @hpk42 will have a final look then (@all so please do not merge yet :)

**Pull request merged**